cmd/fluent-bit/loki_test.go

```diff
@@ -5,30 +5,25 @@ import (
 	"testing"
 	"time"
 
+	lokimodel "github.com/grafana/loki/model"
```

Comment:
It seems unfortunate to mix both model packages. Perhaps we should remove references to the prometheus pkg `model` outside of the new loki `model` pkg and define `type LabelSet model.LabelSet` there.

Then we don't need to import two different model packages. As a bonus, it also gives us a layer of indirection that we can use in the future if we need to easily refactor the label type.